openerp/api.py

```diff
@@ -903,6 +903,15 @@ def norecompute(self):
         finally:
             self.all.recompute = tmp
 
+    def clear_recompute_list(self):
+        del self.all.recompute_list[:]
+
+    def extend_recompute_list(self, items):
+        self.all.recompute_list += items
+
+    def get_recompute_list(self):
+        return self.all.recompute_list
+
```

Comment:
The API looks verbose to me. Why don't you make a property to replace `get_recompute_list` and `extend_recompute_list`?

    @property
    def recompute_old(self):
        return self.all.recompute_old

    def clear_recompute_old(self):
        del self.all.recompute_old[:]